Makefile

```diff
@@ -20,13 +20,13 @@ REGISTRY_AND_USERNAME ?= $(REGISTRY)/$(USERNAME)
 PROTOBUF_GRPC_GATEWAY_TS_VERSION ?= 1.2.1
 TESTPKGS ?= ./...
 NODE_BUILD_ARGS ?=
-PROTOBUF_GO_VERSION ?= 1.33.0
-GRPC_GO_VERSION ?= 1.3.0
-GRPC_GATEWAY_VERSION ?= 2.19.1
+PROTOBUF_GO_VERSION ?= 1.34.1
+GRPC_GO_VERSION ?= 1.4.0
+GRPC_GATEWAY_VERSION ?= 2.20.0
```

Comment:
This Gateway version might have issues. But maybe it works if you bump the gateway version in the `go.mod`...

Please run `make generate generate-frontend` too.

Reply:
Done! Nothing changed.

Reply:
it doesn't work. need to change this line manually yo 2.19.1. then the `make generate-frontend && make generate`.